consensus/state_sync/server.go

```diff
@@ -21,50 +21,52 @@ type StateSyncServerModule interface {
 }
 
 func (m *stateSync) HandleStateSyncMetadataRequest(metadataReq *typesCons.StateSyncMetadataRequest) error {
-	serverNodePeerId, err := m.GetBus().GetConsensusModule().GetCurrentNodeAddressFromNodeId()
+	consensusMod := m.GetBus().GetConsensusModule()
+	serverNodePeerId := m.GetBus().GetConsensusModule().GetNodeAddress()
+
+	clientPeerAddress := metadataReq.PeerAddress
```

Comment:
Can you add a `CONSOLIDATE(#347): Integrate with LibP2P` next to `PeerAddress` in the protobuf?